### ISSUE TITLE
any hyper params suggestions to reproduce the accuracy of the official released ckpt

### ISSUE THREAD

**Issue by AmberzzZZ** [mkturkcan/DART#5]
hi DART team, nice work.
I would politely ask about the specific training settings of the student backbone distillation.
Taking the RepViT-M2.3 for example, what are the settings of training hypers like epochs, learning rate, lr schedule etc.

**Comment by mkturkcan** [comment]
Of course,
https://github.com/mkturkcan/DART/blob/main/scripts/distill.py the distillation parameters are provided in the distillation script itself. We use the 8xH100 scripts provided for the published checkpoints. I will update the README accordingly.
